reproducibility_project/src/engines/cassandra/project.py

```diff
@@ -17,6 +17,255 @@ def __init__(self):
         self.ff_fn = self.data_dir / "forcefield.xml"
 
 
+@Project.label
+def is_cassandra(job):
+    return job.sp.simulation_engine == "cassandra"
+
+@Project.operation
+@Project.post(cassandra_complete)
+@directives(omp_num_threads=4)
+def run_cassandra(job):
+    ""
+
+    import foyer
+    import mbuild as mb
+    import mosdef_cassandra as mc
+    import unyt as u
+    from mbuild.formats.xyz import read_xyz
+
+    from reproducibility_project.src.molecules.methane_ua import MethaneUA
+    from reproducibility_project.src.molecules.pentane_ua import PentaneUA
+    from reproducibility_project.src.molecules.system_builder import (
+        construct_system,
+    )
+
+
+    compound_dict = {
+            "methaneUA": MethaneUA(),
+            "pentaneUA": PentaneUA(),
+            "benzeneUA": BenzeneUA(),
+            "waterSPC/E": mb.lib.molecules.water.WaterSPC(),
+            "ethanolAA": mb.load("CCO", smiles=True),
+            }
+    molecule = job.sp.molecule
+
+    compound = compound_dict[molecule]
+
+    ensemble = job.sp.production_ensemble
+
+    Nliq = job.sp.N_liquid
+    Nvap = job.sp.N_vap
+    N = Nliq+Nvap
+
+    filled_boxes = construct_system(job.sp)
+
+    liqbox_filled = filled_boxes[0]
+    if ensemble == "GEMC-NVT":
+        vapbox_filled = filled_boxes[1]
+
+    ffname = job.sp.forcefield
+    ff = get_ff(ffname) # make this function exist
+    structure = ff.apply(compound)
+
+    T = job.sp.temperature * u.K
+    P = job.sp.pressure * u.kPa
+
+
+    species_list = [structure]
+    cutoff_dict = {
```

Comment:
This information is actually part of the statepoint for each job now.

`job.sp.r_cut`

Reply:
Though these are in nm and need to be converted for your case

Reply:
I fixed it and should push the changes soon.